src/modules/components/member/memberDataListItem/memberDataListItem.tsx

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useState } from 'react';
+import { getAddress } from 'viem';
+import { useAccount } from 'wagmi';
+import { DataList, Heading, NumberFormat, Tag, formatterUtils, type IDataListItemProps } from '../../../../core';
+import { truncateEthAddress } from '../../../../utils/truncateEthereumAddress';
+import { MemberAvatar } from '../memberAvatar';
+
+export interface IMemberDataListItemProps extends IDataListItemProps {
+    /**
+     * Whether the member is a delegate of current user or not.
+     */
+    isDelegate?: boolean;
+    /**
+     * The number of delegations the member has from other members.
+     */
+    delegationCount?: number;
+    /**
+     * The total voting power of the member.
+     */
+    votingPower?: number;
+    /**
+     * ENS name of the user.
+     */
+    ensName?: string;
+    /**
+     * 0x address of the user.
+     */
+    address?: string;
```

Comment:
At least the `address` should be required, there's probably no use-case for rendering a member-data-list-item without an address
```suggestion
    address: string;
```